### DISCUSSION TITLE
コードブロック内のコメント(#)が見出しとして認識されてしまう＆空行を前後に加えても解消しない

### DISCUSSION THREAD

**Discussion by c60evaporator** [increments/qiita-discussions#223] [要望・アイデア・不具合]
https://qiita.com/c60evaporator/items/2b7a2820d575e212bcf4

上記の記事に関しまして、コードブロック内の#（Python等でコメント用に使用）が、右の目次で見出しと認識されてしまうようです。

<img width="800" alt="スクリーンショット 2022-05-15 14 44 38" src="https://user-images.githubusercontent.com/59557625/168459310-44e31c38-8ca7-4f0a-af5a-a59830edeb58.png">

他の方が報告されていた同様の不具合 #178 や #202 では

「コードブロックの前後に空行を加えれば正常に見出しが認識されるようになる」

との記載がございましたが、私のケースでは以下のように前後に空行を加えていても、見出しの誤認識が解消されないようです。

<img width="681" alt="スクリーンショット 2022-05-15 15 03 50" src="https://user-images.githubusercontent.com/59557625/168459612-df585e0f-c266-471f-a21f-16ae8bdf71e2.png">

ご調査頂けますと幸いです。

**Comment by tanaka-asami415** [comment]
@c60evaporator 
ご報告いただきありがとうございます！

確認させていただいたところ、ファイル名に半角スペースが入っているのが原因かと思われます。
`Training API + 旧指定方法(early_stopping_rounds)での実装法`
こちらのファイル名からスペースを取り除いて、お試しいただけないでしょうか🙏

原因としましては、こちらと同様に新しいMarkdownパーサーで発生しているバグになります。
#178 #202 

今後修正予定ですが、不具合修正にお時間をいただいております🙇
それまではお手数ですが、上記の一時的な解決方法をご対応いただけますと幸いです🙏

**Comment by kyntk** [comment]
@c60evaporator
目次の不具合修正が完了いたしました！
お待たせして申し訳ございませんでした 🙇

今後はファイル名に半角スペースを入れていただいても大丈夫です！

**Reply by c60evaporator** [reply]
ありがとうございます！
